draft-ietf-rats-msg-wrap.md

```diff
@@ -532,26 +532,26 @@ lication/signed-corim+cbor"
    03                                   # unsigned(3)
 ~~~
 
-## CBOR Collection
+## CBOR-encoded Collection
 
-The following example is a CBOR collection that assembles conceptual messages from three attesters: Evidence for attesters A and B and Attestation Results for attester C.
-It is given an explicit collection type using the URI form.
+The following example is a CBOR-encoded Collection CMW that assembles conceptual messages from three attesters: Evidence for attesters A and B and Attestation Results for attester C.
+It is given an explicit `"cmwc_t"` using the URI form.
```

Comment:
You use `cmwc_t` and `__ cmwc_t` interchangeably, is it on purpose?

Reply:
Fixed in [4742871](https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/pull/181/commits/47428716fb1c9c8c543a8eed16ce07ce2146d73f)